### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by snicoll**

### CODE REVIEW

File: src/main/java/io/spring/initializr/generator/language/Annotable.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  *
  * @author Andy Wilkinson
  */
-public interface Annotatable {
+public interface Annotable {
```
Comment (wilkinsona) [inline src/main/java/io/spring/initializr/generator/language/Annotable.java]: I think this should be `Annotatable`: https://en.oxforddictionaries.com/definition/annotatable

File: src/main/java/io/spring/initializr/generator/Dependency.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
  *
  * @author Andy Wilkinson
  */
-public class Dependency {
+public class Dependency implements Comparable<Dependency> {
```
Comment (wilkinsona) [inline src/main/java/io/spring/initializr/generator/Dependency.java]: I think this makes sense. I was holding off doing it as I hadn't convinced myself that all users of `Dependency` would want the same ordering. We can always pull it back out again though.

Comment (snicoll) [reply src/main/java/io/spring/initializr/generator/Dependency.java]: That makes sense. Hopefully we want dependencies to be ordered the exact same way in every build system.

**Comment by snicoll** [comment]
@wilkinsona thanks for the feedback. I've pushed force a new proposal.

**Comment by wilkinsona** [comment]
Thanks. Merged into master.

**Pull request closed without merging**